packages/plotly/test_plotly.py

```diff
@@ -0,0 +1,10 @@
+from pyodide_test_runner import run_in_pyodide
+
+
+@run_in_pyodide(packages=["plotly"])
```

Comment:
you will probably need:
```suggestion
@pytest.mark.xfail_browsers(node="plotting doesn't work in node")
@run_in_pyodide(packages=["plotly"])
```